tlog-tiles.md

```diff
@@ -154,6 +159,60 @@ compatibility concerns. If they do, they SHOULD also serve entry bundles, and
 MUST serve the differently-encoded entries at a different URL from the specified
 entry bundle URL.
 
+### Pruning
+
+In some log applications, such as [Certificate Transparency][], entries expire
+and are replaced with renewed versions. As this happens, the total size of the
+log grows, even if the unexpired subset remains fixed. To mitigate this, a log
+MAY be *pruned*. Pruning makes some prefix of the log unavailable, without
+changing the tree structure.
+
+Logs maintain a *minimum index* value. The minimum index is the index of the
+first log entry that the log publishes. It MUST be less than or equal to the
+tree size of the log's current checkpoint. In an unpruned log, the minimum index
+is zero.
+
+In response to a request for a full tile `<prefix>/tile/<L>/<N>`, or an entry
+bundle `<prefix>/tile/entries/<N>`, the log MAY return a 404 response if the
```

Comment:
```suggestion
bundle `<prefix>/tile/entries/<N>`, the log MAY return a 410 or 404 response if the
```

Requiring 410 would be nicer but would make hosting on e.g. S3 impossible. It'd be nice for logs that have space for that logic in the read path to do this, though.